spec/spec_helper.rb

```diff
@@ -68,19 +68,8 @@ def header_sanitizer(response_header, transient_headers)
   config.ignore_hosts('codeclimate.com') if ENV['CI']
   config.cassette_library_dir = File.join(ManageIQ::Providers::IbmCloud::Engine.root, 'spec/vcr_cassettes')
 
-  config.before_record do |i|
-    replace_token_contents(i) if i.request.uri == "https://iam.cloud.ibm.com/identity/token"
-    vpc_sanitizer(i) if i.request.uri.match?('iaas.cloud.ibm') || i.request.uri.match?('tags.global-search-tagging')
-  end
```

Comment:
The tests passed without this, so I'm not sure why it's needed, unless it's during recording?